src/main/java/com/redislabs/redisai/RedisAI.java

```diff
@@ -350,6 +367,49 @@ public boolean runScript(String key, String function, String[] inputs, String[]
     }
   }
 
+  /**
+   * {@code AI.SCRIPTRUN <key> <function> [TIMEOUT t] INPUTS <input> [input ...] [$ input ...]
+   * OUTPUTS <output> [output ...]}
+   */
+  public boolean runScript(
+      String key, String function, String[] inputs, boolean variadicInputs, String[] outputs) {
+    try (Jedis conn = getConnection()) {
+      String[] args = scriptRunFlatArgs(key, function, inputs, variadicInputs, outputs);
+      return sendCommand(conn, Command.SCRIPT_RUN, args).getStatusCodeReply().equals("OK");
+    } catch (JedisDataException ex) {
+      throw new RedisAIException(ex);
+    }
+  }
+
+  protected static String[] scriptRunFlatArgs(
```

Comment:
Is there a reason not to use: https://github.com/RedisAI/JRedisAI/blob/master/src/main/java/com/redislabs/redisai/Script.java#L151 ?

Reply:
@filipecosta90 We are not using anything from Script. So I didn't feel the necessity to use the Script.

Also, see that, `Script.scriptRunFlatArgs` is almost invalidated because of new version. But we can't just remove it because it breaks backward compatibility. I tried to avoid introducing more such cases.

Reply:
> Also, see that, Script.scriptRunFlatArgs is almost invalidated because of new version.
Please explain what is invalid. Given we don't support timeout on it I believe it will still be valid, and we only need to add the variadic addition.

Reply:
The new scriptRunFlatArgs method (with variadicInputs param) is able to serve the existing one. Likewise, when we will add timeout param, it would be able to serve the previous two. By invalidated I meant this scenario where new method is able serve existing/old methods as well.